process/peer/validatorsProvider_test.go

```diff
@@ -732,96 +732,97 @@ func TestValidatorsProvider_GetAuctionList(t *testing.T) {
 		response := vp.GetAuctionList()
 		require.Empty(t, response)
 	})
-
```

Comment:
Add a todo here to FIX the test or remove it

Reply:
There is no fix for this actually.
It is the first API version; which will be completely removed.
I think it used a map while processing causing non-deterministic results